.azure-pipelines/compliance/CredScanSuppressions.json

```diff
@@ -0,0 +1,13 @@
+{
+    "tool": "Credential Scanner",
+    "suppressions": [
+        {
+            "folder": "node_modules\\",
+            "_justification": "No need to scan external node modules."
+        },
+        {
```

Comment:
remove this entry because it's specific to functions. Also make sure to queue a build with `ENABLE_COMPLIANCE` set to true to see if this kind of stuff passes